Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM debian:stable-slim
+
+ENV APT_GET_UPDATE 2019-01-01
+
+WORKDIR /app
+COPY . /app
```

Comment:
Since we always mount the working directory anyways this COPY is rather useless.